src/main/java/app/service/comment/CommentService.java

```diff
@@ -0,0 +1,18 @@
+package app.service.comment;
+
+import app.dto.comment.CommentRequestDto;
+import app.dto.comment.CommentResponseDto;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+
+public interface CommentService {
+    CommentResponseDto save(CommentRequestDto commentRequestDto, Long userId, Long taskId);
+
+    List<CommentResponseDto> findByUserId(Long userId, Pageable pageable);
+
+    List<CommentResponseDto> findByTaskId(Long taskId, Pageable pageable);
```

Comment:
On the service layer methods are named "get by ..." because they return an object if it exists. The methods should be named "find ..." on the repository level if they return optional

Reply:
Agree.